### ISSUE TITLE
训练时 报错

### ISSUE THREAD

**Issue by JWLe666** [PaddlePaddle/PaddleOCR#3435]
![1627356142(1)](https://user-images.githubusercontent.com/49588285/127090357-d0994416-2d4b-4d01-8309-cc0e461210da.jpg)
请问这种都训练了一轮 为什么还会退出

我的yaml
`Global:
  use_gpu: true
  epoch_num: 72
  log_smooth_window: 20
  print_batch_step: 10
  save_model_dir: ./1/
  save_epoch_step: 3
  # evaluation is run every 2000 iterations
  eval_batch_step: [0, 1000]
  cal_metric_during_train: True
  pretrained_model: null
  checkpoints: null
  save_inference_dir : null
  use_visualdl: False
  infer_img: doc/imgs_words_en/word_10.png
  # for data or label process
  character_dict_path: ppocr/utils/en_dict.txt
  character_type: ch
  max_text_length: 1000
  infer_mode: False
  use_space_char: True
  save_res_path: ./1/predicts_ic15.txt

Optimizer:
  name: Adam
  beta1: 0.9
  beta2: 0.999
  lr:
    learning_rate: 0.0005
  regularizer:
    name: 'L2'
    factor: 0

Architecture:
  model_type: rec
  algorithm: CRNN
  Transform:
  Backbone:
    name: MobileNetV3
    scale: 0.5
    model_name: large
  Neck:
    name: SequenceEncoder
    encoder_type: rnn
    hidden_size: 96
  Head:
    name: CTCHead
    fc_decay: 0

Loss:
  name: CTCLoss

PostProcess:
  name: CTCLabelDecode

Metric:
  name: RecMetric
  main_indicator: acc

Train:
  dataset:
    name: SimpleDataSet
    data_dir: ./ch4/
    label_file_list: [./ch4/train_list1.txt]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False  
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 100]      
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: True
    batch_size_per_card: 1
    drop_last: True
    num_workers: 0
    use_shared_memory: False
    
Eval:
  dataset:
    name: SimpleDataSet
    data_dir: ./ch5/
    label_file_list: [./ch5/train_list1.txt]
    transforms:
      - DecodeImage: # load image
          img_mode: BGR
          channel_first: False
      - CTCLabelEncode: # Class handling label
      - RecResizeImg:
          image_shape: [3, 32, 100]      
      - KeepKeys:
          keep_keys: ['image', 'label', 'length'] # dataloader will return list in this order
  loader:
    shuffle: False
    drop_last: False
    batch_size_per_card: 1
    num_workers: 0
    use_shared_memory: False
    `

我的数据集
![1627356353(1)](https://user-images.githubusercontent.com/49588285/127090552-979795e3-6f4e-4344-b69e-9d75c2f8daca.jpg)

**Comment by WenmuZhou** [comment]
debug下面有报错日志，贴出来看看

**Comment by JWLe666** [comment]
我没有debug文件夹  我一用--log_dir 'debug/'参数  就会报错
usage: train.py [-h] [-c CONFIG] [-o OPT [OPT ...]]
train.py: error: unrecognized arguments: -log_dir debug/

**Comment by WenmuZhou** [comment]
你好，用最新的代码还会出现这个问题吗
